packages/common/src/logger/logger.ts

```diff
@@ -72,7 +73,13 @@ export class Logger {
           return Object.prototype.toString.apply(input) === '[object Object]';
         }
       },
-    });
+    } as Pino.LoggerOptions;
+
+    if (toFile) {
+      this.pino = Pino(options, Pino.destination('./logger.log'));
```

Comment:
Left log file name as nondescript incase want to use in another package.

Reply:
log files should be rotated

Reply:
I've updated so you can specify log file name and dir.